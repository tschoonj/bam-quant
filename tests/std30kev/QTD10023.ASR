File: C:\xrfdaten\2014KW30\chris\std30kev\std10023.spe
Start: 22.07.2014 09:55:53
Stop: 22.07.2014 09:57:33
BESSY: 298.1715946 298.2127811
IONI: 0.000000000020556 2.04281E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 98203
Run: 100000
Dead: 2
ICR: 240
OCR: 0
Accumulated BESSY: 59703.224209
Accumulated Ioni: 4.11721474E-09
N Accumulation: 200
Normfactor:  4.9465422e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: V
exp_x 92.7203979492188 92.7204
exp_y 150.465789794922 150.4658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  23   1      4.950         8.         5.        .95
