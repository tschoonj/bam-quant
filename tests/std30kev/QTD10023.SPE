$DATA:
0 2000
23141
14807
6395
1724
349
48
6
0
0
1
0
0
0
0
0
0
0
21
19
20
21
21
20
23
17
20
15
16
15
17
16
17
21
17
17
24
14
18
14
16
13
16
11
7
16
15
10
19
14
10
12
17
13
11
15
11
13
9
10
16
19
11
20
8
11
7
13
11
6
8
12
8
10
9
11
7
12
20
12
12
10
12
9
10
11
10
9
9
8
11
13
8
6
8
11
10
7
15
16
4
15
6
9
11
11
9
5
13
14
10
5
6
7
9
12
15
9
15
14
8
9
8
10
15
11
5
7
10
11
12
15
15
6
13
10
7
9
12
10
12
11
13
11
12
15
16
14
8
10
4
11
9
15
6
16
12
8
10
12
8
12
2
9
8
7
9
4
10
12
6
10
14
11
10
10
8
16
7
14
7
8
13
6
5
6
7
3
5
11
9
7
6
15
8
9
6
7
6
9
1
5
5
5
3
6
5
7
8
9
6
5
3
0
5
2
4
6
2
7
5
3
5
3
4
3
4
2
2
1
3
4
3
3
1
1
3
2
1
4
3
1
1
2
5
3
3
3
0
3
2
1
0
1
2
2
4
4
1
2
3
0
1
2
1
0
2
0
3
4
4
3
1
4
0
3
3
3
2
1
1
0
2
1
2
1
0
2
4
1
2
2
2
3
0
1
0
0
0
2
0
0
0
0
1
3
1
2
1
1
2
1
5
0
3
7
5
4
5
3
6
8
3
1
4
0
1
5
2
0
4
2
3
1
1
2
0
0
0
1
0
3
2
2
1
0
0
3
2
1
0
3
0
3
1
0
0
2
0
0
2
1
0
2
0
0
1
1
2
4
4
2
1
0
2
0
2
5
0
0
0
1
1
2
1
2
1
1
2
3
2
0
1
1
1
1
1
1
1
0
0
0
1
0
2
1
1
4
0
1
1
0
0
1
1
0
1
3
1
0
0
0
0
0
0
1
0
0
1
0
0
0
1
0
0
1
0
1
1
1
1
1
0
2
1
0
1
2
0
1
0
0
1
2
1
1
0
1
1
0
0
0
0
0
1
1
1
1
2
0
2
3
2
1
0
1
0
0
1
1
0
0
0
1
2
1
0
1
0
1
0
1
0
0
1
0
0
3
1
2
3
1
0
2
0
0
1
2
1
2
2
1
1
0
2
2
1
1
0
0
0
0
1
1
0
1
2
2
4
1
2
0
0
2
0
1
0
0
2
0
4
0
2
1
3
1
1
0
2
1
2
4
2
3
3
1
0
1
0
2
6
1
1
3
3
1
1
2
3
1
1
0
2
1
0
0
2
2
0
2
2
1
0
3
0
4
2
1
1
1
3
2
0
4
2
3
1
2
1
1
1
1
2
2
1
2
1
2
1
4
3
1
3
5
4
3
2
2
1
2
3
4
4
2
6
2
3
2
2
3
1
3
1
1
3
2
3
2
3
0
4
5
1
4
1
3
5
3
2
1
2
5
6
4
3
5
4
4
1
2
0
4
2
3
2
2
5
7
2
1
4
3
0
2
2
2
5
5
3
5
3
3
5
3
3
1
2
4
2
2
3
2
4
2
4
2
3
1
2
4
4
3
4
0
6
5
2
3
5
5
4
4
4
2
5
3
5
6
6
3
1
5
1
3
3
5
4
2
4
3
2
2
4
4
4
3
5
4
2
3
5
2
4
5
2
2
6
3
4
4
7
1
5
6
2
4
7
4
4
4
4
5
3
3
5
9
3
10
13
12
15
21
29
37
58
67
71
114
105
100
115
104
107
120
86
82
69
54
49
38
19
17
15
5
7
4
9
4
4
6
4
8
3
4
5
6
3
7
6
6
5
6
12
16
19
22
24
33
41
45
65
40
55
55
48
42
42
33
24
18
14
14
7
10
5
9
2
7
10
3
7
3
2
7
6
4
4
5
3
6
6
6
11
7
7
13
25
23
27
32
35
43
39
42
47
51
57
50
52
39
48
44
48
37
33
24
25
26
22
16
14
17
12
8
11
6
5
4
5
9
4
7
3
8
7
4
4
10
4
7
2
1
3
6
2
2
6
6
3
5
5
3
3
3
7
6
5
10
16
16
16
16
21
26
13
22
12
16
9
11
14
8
7
5
3
3
4
6
11
6
8
6
6
3
6
1
4
7
6
4
7
4
0
5
5
6
4
5
3
4
5
4
6
7
7
6
10
6
12
16
17
14
12
13
13
14
10
8
9
8
4
6
4
7
7
2
2
3
1
4
7
2
9
4
4
0
10
2
5
3
4
2
4
4
3
1
5
4
2
8
3
2
3
3
7
1
3
5
3
5
2
2
1
3
4
3
2
1
0
2
1
5
3
4
7
3
2
1
2
4
1
3
2
3
1
5
3
3
1
3
4
1
1
3
4
5
2
4
3
6
2
3
4
3
2
4
8
5
1
2
2
7
1
5
4
3
5
2
1
6
4
5
1
1
4
2
4
6
4
6
2
2
1
2
7
4
2
4
5
3
8
4
3
5
2
4
6
3
2
9
4
2
0
2
3
1
7
4
6
7
4
2
3
3
4
2
3
5
2
3
1
1
4
2
2
3
3
3
4
3
1
2
2
2
1
4
8
7
7
3
3
3
2
0
4
5
3
3
0
2
0
2
2
4
6
2
3
4
3
3
7
6
3
2
0
2
3
2
5
5
2
4
3
3
4
2
4
2
2
2
4
4
2
5
6
1
8
3
1
2
2
3
4
6
5
5
1
1
6
1
1
0
3
5
6
2
3
4
5
3
1
4
5
12
2
4
3
1
2
6
1
7
5
7
3
10
8
12
8
16
10
9
12
10
8
7
21
11
14
23
13
11
22
26
8
17
11
16
13
11
6
8
3
5
4
3
3
2
5
3
3
0
4
3
3
1
7
4
2
1
6
3
8
3
2
6
2
4
6
2
2
4
3
4
3
4
4
7
5
1
4
9
4
5
7
7
3
3
3
5
5
6
2
3
3
2
0
1
2
4
7
3
4
4
3
4
6
4
5
5
2
5
6
6
2
2
1
6
6
4
5
6
6
5
6
8
8
8
9
4
5
6
3
4
6
9
7
6
2
4
9
4
4
7
9
9
7
9
3
6
11
1
4
3
2
4
3
8
10
4
5
8
8
5
10
10
6
3
13
2
3
4
6
2
10
9
9
8
10
8
7
14
10
8
13
8
14
4
13
6
13
13
16
7
13
13
17
6
9
13
11
11
6
8
10
5
13
12
7
14
5
9
11
8
13
8
8
10
13
8
11
5
11
9
15
13
18
11
7
12
11
12
12
10
6
10
10
12
14
10
15
13
12
11
12
7
15
12
14
12
10
9
14
6
7
9
11
11
14
11
14
14
11
7
13
9
4
9
13
7
11
13
14
11
7
14
11
11
11
9
16
18
16
17
20
13
9
21
15
11
19
15
9
9
15
15
11
15
18
15
17
14
14
13
9
12
14
18
8
9
25
15
17
20
23
26
17
17
24
22
17
21
26
31
20
19
26
20
24
27
24
25
24
21
28
27
35
22
27
25
24
28
33
37
28
29
20
28
25
27
32
34
24
32
26
31
38
40
38
34
35
38
27
38
32
37
31
45
44
42
45
43
39
53
37
32
36
48
69
54
46
70
61
64
58
50
60
55
78
66
62
66
63
65
77
72
57
66
66
75
66
79
62
87
80
74
66
87
75
64
79
77
65
81
84
70
79
68
73
72
78
75
69
48
71
65
63
45
52
56
62
70
50
69
54
46
49
44
53
44
52
50
52
47
43
36
31
47
39
55
43
35
41
50
39
39
39
41
35
41
51
31
38
35
50
30
39
37
36
41
39
47
28
43
40
40
42
32
31
40
34
26
31
33
40
38
40
37
28
23
33
28
25
31
39
38
25
33
37
40
31
40
55
47
40
57
54
56
51
65
69
49
66
76
84
78
89
76
87
71
59
67
67
83
72
83
81
64
81
57
69
49
51
48
47
51
40
28
35
28
21
26
20
21
20
19
9
7
8
6
7
0
4
1
1
0
3
0
2
2
0
0
2
1
1
0
0
0
1
2
0
1
1
0
1
1
0
1
0
2
0
0
0
0
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10023.spe
Start: 22.07.2014 09:55:53
Stop: 22.07.2014 09:57:33
BESSY: 298.1715946 298.2127811
IONI: 0.000000000020556 2.04281E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 98203
Run: 100000
Dead: 2
ICR: 240
OCR: 0
Accumulated BESSY: 59703.224209
Accumulated Ioni: 4.11721474E-09
N Accumulation: 200
Normfactor:  4.9465422e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: V
 
exp_x 92.7203979492188 92.7204
exp_y 150.465789794922 150.4658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  23   1      4.950         8.         5.        .95
