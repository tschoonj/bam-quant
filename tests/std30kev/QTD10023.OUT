     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:28:43  บ   
     บ  Spectrum: QTD10023.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        2 iterations done    บ   
     บ  ChiSquare =     9.5   last change =   .00%    lambda= 1.E-04   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -42.4 ๑   100.0   บ     -42.4 ๑    97.7   บ   
     บ  GAIN (eV/ch)  บ     20.187 ๑   2.000   บ    20.187 ๑    .416   บ   
     บ det NOISE (eV) บ      206.1 ๑    40.0   บ     206.1 ๑    55.1   บ   
     บ  FANO factor   บ       .040 ๑    .050   บ      .040 ๑    .050   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 V -Ka                                 8. ๑      5.           บ   
     บ    KA1         4.950   1.00000           8. ๑      7.       .9  บ   
     บ              247.303    216.08          18.           8.31E-01  บ   
     บ                                                                 บ   
     บ  2 V -Kb                                11. ๑      6.           บ   
     บ    KB1         5.427   1.00000          11. ๑      7.      1.0  บ   
     บ              270.932    217.01          18.           8.69E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
